cymetric/metrics.py

```diff
@@ -465,6 +465,45 @@ def monthly_electricity_generated_by_agent(elec):
 
 
 del _egdeps, _egschema
+
+
+# Quantity per GigaWattElectric in Inventory [kg/GWe]
+_invdeps = ['ExplicitInventory', 'TimeSeriesPower']
+
+_invschema = [
+    ('SimId', ts.UUID),
+    ('AgentId', ts.INT),
+    ('Time', ts.INT),
+    ('InventoryName', ts.STRING),
+    ('NucId', ts.INT),
+    ('Quantity', ts.DOUBLE)
+]
+
+
+@metric(name='InventoryQuantityPerElectricPower', depends=_invdeps,
+        schema=_invschema)
+def inventory_quantity_per_gwe(inv, power):
+    """Inventory Quantity per GWe metric returns the explicit inventory table
+    with quantity in units of kg/GWe, calculated by dividing the original
+    quantity by the electricity generated at the corresponding simulation and
+    the specific time in TimeSeriesPower metric.
+    """
+    # reference columns used for inventatory normalisation
+    base_index = ['SimId', 'Time']
+
+    # preping the power table to get 1 power per time
+    power.drop(["AgentId"], axis=1, inplace=True)
```

Comment:
If the dataframe is passed by reference, this drops the `AgendId` from the power data frame that was passed into this method, doesn't it?  Do we want to do that?

Reply:
this is correct but not only because it is passed by reference but because of the `inplace`

`power = power.drop(["AgentId"], axis=1)` will not change the the initial `power` data frame

Reply:
I committed a fix for it.